### DISCUSSION TITLE
Why change from named properties to an array?

### DISCUSSION THREAD

**Discussion by kaioenzo** [pdfme/pdfme#607] [General]
It is very easy to make inputs with named properties like was is v4; in v5 with an array, we have to put schema in order, wich was very easy to make a mistake. For me, this is a downgrade and I want to understand this decision.

**Comment by peteward** [comment]
Hi.

I can see from your other comment that you have probably found the reason for the change.

I don't understand why you need to "put the schema in order", and what mistake can this lead to?

The schema now should be guaranteed to be in the correct order and inputs are still keyed
